windows-gaming-driver/src/driver/clipboard.rs

```diff
@@ -147,7 +147,8 @@ impl X11Clipboard {
                         controller.borrow_mut().respond_win_clipboard(Vec::new());
                     }
                 }
-                _ => info!("Unknown XCB event: {}", event.response_type()),//unimplemented!(),
+                PROPERTY_NOTIFY => debug!("Ignoring PROPERTY_NOTIFY event"),
```

Comment:
This should be a trace. We ignore it for a reason (which is not to spam debug log ;) )